Public/Src/Utilities/UnitTests/Executables/TestProcess/Operation.cs

```diff
@@ -240,20 +240,30 @@ public enum SymbolicLinkFlag : uint
         public Type OpType { get; private set; }
 
         /// <summary>
-        /// Path to file
+        /// Path to file. NOTE: This will not be accessable within TestProcess.exe

```

Comment:
>TestProcess [](start = 67, length = 11)

Should this then simply be conditionally compiled bteween the two?

Reply:
good idea. done.